treemodeladmin/static/treemodeladmin/css/index.css

```diff
@@ -2,20 +2,6 @@ header.hasform {
     margin-bottom: 0;
 }
 
-header .button-small {
-    background-color: transparent;
-    color: #fff;
-    border-color: #358c8b;
-}
-
-header .button-small:hover {
-    background: #358c8b;
-}
-
-header .button.bicolor {
-    background: #358c8b;
-}
-
```

Comment:
Unfortunately removing this to fix the UI in 3.x breaks the UI in 2.x:

<img width="286" alt="image" src="https://user-images.githubusercontent.com/654645/208972843-b5ac626b-0371-4d00-a29f-b0c0301bc3b3.png">

Reply:
🙈

Reply:
Fixed in [2b34ec1](https://github.com/cfpb/wagtail-treemodeladmin/pull/34/commits/2b34ec11166a67ad32a8294e8bb9cd74b9d37e2f).